src/main/groovy/com/adarshr/gradle/testlogger/TestLoggerExtensionProperties.groovy

```diff
@@ -84,4 +84,9 @@ abstract class TestLoggerExtensionProperties {
      * Whether simple class names should be used for displaying test suites. Defaults to false.
      */
     Boolean showSimpleNames
+
+    /**
+     * Whether only slow tests should be shown. Defaults to true.
```

Comment:
This should default to false. We're not showing _only_ slow tests all the time.

Reply:
Oh my mistake. Default is set to false now.